vulnerabilities/data_source.py

```diff
@@ -87,17 +88,16 @@ class Advisory:
 
     summary: str
     vulnerability_id: Optional[str] = None
-    impacted_package_urls: Iterable[PackageURL] = dataclasses.field(default_factory=list)
-    resolved_package_urls: Iterable[PackageURL] = dataclasses.field(default_factory=list)
+    affected_packages_with_patched_package: List[
```

Comment:
Name problem 1

Reply:
Could you survey the words use to depict affected/patched in other tools? 
- kaybee/vulas
- dependencycheck and dependency track (and OWASP)
- in general all the data source we use
...
with these listed here we can make a good pick together!

Reply:
The general renaming should be tracked in a separate ticket IMHO, not here.

Beyond this, what about `affected_packages` or just `impacted_packages`?